PSReadLine/YankPaste.vi.cs

```diff
@@ -70,6 +70,26 @@ private void SaveLinesToClipboard(int lineIndex, int lineCount)
             _clipboard.LinewiseRecord(_buffer.ToString(range.Offset, range.Count));
         }
 
+        /// <summary>
+        /// Removes a portion of text from the buffer
+        /// and saves it to the clipboard in order to support undo.
+        /// </summary>
+        /// <param name="start"></param>
+        /// <param name="count"></param>
+        /// <param name="instigator"></param>
+        /// <param name="arg"></param>
+        private void RemoveTextToClipboard(int start, int count, Action<ConsoleKeyInfo?, object> instigator = null, object arg = null)

```

Comment:
`_clipboard` is not a good variable name IMO. It's actually VI specific, but the name is too general.
How about we rename this method to be `RemoveTextAndSaveToViRegister`?

Reply:
I chose `RemoveTextToViRegister` if that is OK for you.

Reply:
I also introduced an extra commit to rename `_clipboard` to `_viRegister`.
Please, let me know if you want this change or not.